notebooks/book1/02/change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.    ax[2].set_ylabel("$Density$")
</pre><p>Rename it to "Frequency" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
![save_fig](https://user-images.githubusercontent.com/87682045/169585888-cffc31ab-5d91-473b-9d2d-6b4d9ab0be02.PNG)

Reply:
![figures](https://user-images.githubusercontent.com/87682045/169586276-d33c3648-322e-4ad0-855c-cfcef6e50042.PNG)